VERSION

```diff
@@ -1 +1 @@
-1.3.0
+1.4.0
```

Comment:
Please don't bump the version. If you want to add a CHANGELOG entry, use `## master / unreleased` at the top of the file.